magda-web-client/src/Components/Common/DataPreviewMap.js

```diff
@@ -18,8 +18,18 @@ export const defaultDataSourcePreference = [
 ];
 
 export const isSupportedFormat = function(format) {
+    const dataSourcePreference = defaultDataSourcePreference.map(
+        preferenceItem => {
+            if (typeof preferenceItem === "string") return preferenceItem;
```

Comment:
Could we put some `{}`s around this one? Makes it a bit hard to see what's going on :)

Reply:
Updated~